morpheus/messages/multi_inference_message.py

```diff
@@ -54,11 +54,11 @@ def get_input(instance, name: str):
     AttributeError
         If input name does not exist in message container.
     """
-    if (name not in instance.tensors):
+    try:
+        return instance.get_tensors()[name]
```

Comment:
Lets add a `get_tensor(name)` method to avoid copying all of the tensors even if they arent needed.